red_queen/games/applications/run_bv.py

```diff
@@ -13,9 +12,30 @@
 
 from red_queen.games.applications import backends, run_qiskit_circuit
 
+import random
+
+import requests
+
+import numpy as np
 
 QASM_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "qasm")
-SECRET_STRING = "110011"
+
+
+@pytest.fixture
+def get_num_qubits(request):
+    user_params = request.config.getoption("--num_qubits")
+    user_params = user_params.split(":")
+    if user_params[0] == "linear":
+        sweep = np.linspace(int(user_params[1]), int(user_params[2]), int(user_params[3]))
+        sweep = [int(i) for i in sweep]
```

Comment:
You can simplify this by specifying the dtype directly:

```suggestion
        sweep = np.linspace(int(user_params[1]), int(user_params[2]), int(user_params[3]), dtype=int)
```
This will return a numpy array with an integer datatype